### PR TITLE
Resolve COR comments in introduction

### PR DESCRIPTION
**Pull request by Venyla**

### CODE REVIEW

File: chapters/introduction.typ [hunk 1]
```diff
@@ -19,17 +19,16 @@ To support the C++ community with its refactoring tasks, new refactoring feature
...
 Concepts are a way of abstracting these constraints. @constraints_and_concepts
 
 Refactoring is a common technique to resolve code smells and improve the internal structure of code without changing its external behavior. @refactoring
 Automated tests often ensure that the correct functionality is retained.
 
 Older versions of integrated development environments (IDE) were implementing support for code analysis and tools like symbol lookup and refactorings themselves.
-// COR And also that every tool had to spend the effort to implement the whole support.
-This led to the problem that new languages only slowly gained adoption, one editor at a time.
+This led to the problem that new languages only slowly gained adoption, one editor at a time and also that every tool had to spend the effort to implement the whole support.
+Another downside was that every tool had to spend the effort to implement the whole support on their own.
```
Comment (jeremystucki) [inline chapters/introduction.typ:31]:
```suggestion
This led to the problem that new languages only slowly gained adoption, one editor at a time, each of them having to spend the effort to implement support for it.
```

File: chapters/introduction.typ [hunk 2]
```diff
@@ -51,9 +50,21 @@ Therefore, it would be nice to have some support for new language features like
...
-The goal of this semester project is to come up with new ideas for refactor operations specific to concepts and to implement some of them.
-Ideally, they should be submitted upstream as a pull request to clangd.
+This section describes the goals of this project according to the task assignment @assignment.
+Additionally, parts were added to give the project more structure, as this project is more explorative than usual.
+
+The goal of this semester project is to come up with new ideas for refactoring operations specific to parameter type constraints and to implement some of them.
+It should be checked if currently existing refactorings can be applied to concepts.
+This may already be implemented in the currently available tooling.
+Ideally, new refactorings should be submitted upstream as a pull request to clangd.
+This is done to support the C++ community while coding as well as helping the LLVM project grow.
```
Comment (jeremystucki) [inline chapters/introduction.typ:60]:
```suggestion
This is done to support the C++ community as well as helping the LLVM project grow.
```

File: chapters/introduction.typ [hunk 3]
```diff
@@ -51,9 +50,21 @@ Therefore, it would be nice to have some support for new language features like
...
+
+In addition to this, research will also be carried out to determine how the clangd language server is communicating with the development tools.
+This also includes documenting the basic knowledge needed to understand the analysis of it.
```
Comment (jeremystucki) [inline chapters/introduction.typ:63]:
```suggestion
This also includes documenting the basic knowledge needed to understand it.
```

File: chapters/introduction.typ [hunk 4]
```diff
@@ -51,9 +50,21 @@ Therefore, it would be nice to have some support for new language features like
...
+
+For the implementation itself, it needs to be clear where the code needs to be added, how it should be tested, what the coding guidelines are, and how it can be contributed.
+Each implemented refactoring feature should be documented for how it can be used and how it transforms the code.
```
Comment (jeremystucki) [inline chapters/introduction.typ:66]:
```suggestion
Each implemented refactoring feature should be documented, including is usage and how it transforms the source code. 
```

File: main.typ [hunk 5]
```diff
@@ -146,12 +159,4 @@ It also contains the final version of the code written during this project (@sou
...
-// #showSourceFile("Test Project", "test-project-source-code/AbbreviateFunctionTemplate.cxx")
-
-// - Relevante Anhänge
-// - Meeting Protokolle
-// - Vereinbarungen
-// - (SA/BA): Persönliche Berichte (0.5-1 Seite)
-// - Entwicklerdokumentation
-// - User Dokumentation
-// - Sonstige Protokolle (z.B. von Usability Tests)
+// #showSourceFile("Test Project", "test-project-source-code/AbbreviateFunctionTemplate.cxx")
```
Comment (jeremystucki) [inline main.typ:162]: Missing newline

File: main.typ [hunk 6]
```diff
@@ -116,6 +116,19 @@ It also contains the final version of the code written during this project (@sou
 == Personal Report — Vina Zahnd <report_vina>
 #include "chapters/personalReportVina.typ"
 
+== Assignment <assignment>
+
+// TODO use assignment with signature for print version (!do not upload to github)
```
Comment (jeremystucki) [inline main.typ:121]: ehmmm wie machemer das? seli luege dass mers au mitem source code lokal chan builde?

File: chapters/introduction.typ (hunk 1 above)
Comment (Venyla) [reply chapters/introduction.typ:31]: wenn de satz wiedre usechunt isch de comment ned resolved:
// COR And also that every tool had to spend the effort to implement the whole support.

File: main.typ (hunk 6 above)
Comment (Venyla) [reply main.typ:121]: Joa aso wennd gad eh idee hesch, susch dümmer eifach bim drucke churz s andere pdf inlcude

File: main.typ (hunk 5 above)
Comment (Venyla) [reply main.typ:162]:
```suggestion
// #showSourceFile("Test Project", "test-project-source-code/AbbreviateFunctionTemplate.cxx")

```

File: chapters/introduction.typ (hunk 1 above)
Comment (jeremystucki) [reply chapters/introduction.typ:31]: "each of them having to spend the effort to implement support for it." langet doch ned?

File: main.typ (hunk 6 above)
Comment (jeremystucki) [reply main.typ:121]: ja de code isch ja nur dinne wenns github buildet, muen luege dass mer das lokal au chend. mache suscht es chlises script

Comment (Venyla) [reply main.typ:121]: ahh ja voll, mümr fasch. Chönd d files au eifach abelade und lokal die lade.

**Review by jeremystucki — APPROVED** [review]

**Pull request merged**